Build frameworks.

shoumikhin opened the pull request:
Summary: .

Reviewed By: huydhn

Differential Revision: D54791338

huydhn commented on .github/workflows/ios-build.yml:
```diff
@@ -0,0 +1,46 @@
+name: Build ExecuTorch demo apps
```
I think let's not create a new workflow if it's not needed.  We can use https://github.com/pytorch/executorch/blob/main/.github/workflows/app-build.yml with `build-ios` as a different job there

shoumikhin committed:
Build frameworks. (#2413)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/2413

bypass-github-export-checks

Reviewed By: huydhn

Differential Revision: D54791338
